.github/gradle/gradle.gradle

```diff
@@ -8,13 +8,12 @@ if (!project.hasProperty('githubCiTesting') && System.env['GITHUB_OUTPUT'] == nu
 
 static Map<String, String> removeCommonPrefix(List<String> list) {
     if (list.size() == 0) {
-        return new HashMap<String, String>()
+        return [:]
     }
 
     def result = new HashMap<String, String>()
```

Comment:
You should replace this with `[:]` as well.

Reply:
Doing that with a def creates a weird map, that is not typed -.-
So I decided this is the better option.